### DISCUSSION TITLE
Welcome to StrongGrid Discussions!

### DISCUSSION THREAD

**Discussion by Jericho** [Jericho/StrongGrid#359] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with StrongGrid.

**Comment by AnubhaVermaWTC** [comment]
Hi @Jericho

**Reply by Jericho** [reply]
Hi and welcome to the community.
